docs/monitoring.md

```diff
@@ -229,10 +229,28 @@ for a running application, at `http://localhost:4040/api/v1`.
     <td>A list of all applications</td>
   </tr>
   <tr>
+     <td></td>
+     <td><code>?status=[completed|running]</code> list only applications in the chosen state</td>
+  </tr>
+  <tr>
+     <td></td>
+     <td><code>?minDate=[date]</code> earliest date/time to list. Examples:
+     <br><code>?minDate=2015-02-10</code>
```

Comment:
actually no, it MUST be `<br>` without a tag, except in the specific case of XHTML, which nobody uses

1. [Mozilla](https://developer.mozilla.org/en/docs/Web/HTML/Element/br)
1. [MSFT](https://msdn.microsoft.com/en-us/library/ms535208(v=vs.85).aspx)

This is the same as the paragraph tag, {{<p>}}.